voidcrew/code/modules/research/designs/biogenerator_designs.dm

```diff
@@ -0,0 +1,19 @@
+///////////////////////////////////
+///////Biogenerator Designs ///////
+///////////////////////////////////
+
+/datum/design/research_notes
+	name = "Research Notes (1000)"
+	id = "research_notes"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass= 16000)
+	build_path = /obj/item/research_notes/loot/small
+	category = list("initial","Organic Materials")
+
+/datum/design/strange_seeds
+	name = "Pack of Strange Seeds"
+	id = "strange_seeds"
+	build_type = BIOGENERATOR
+	materials = list(/datum/material/biomass= 5000)
```

Comment:
```suggestion
	materials = list(/datum/material/biomass = 5000)
```

Reply:
yea it was a quick copy from the biogenerator template updated